tests/testing_utility.py

```diff
@@ -75,6 +77,101 @@ def test_is_node_on_plane(self):
             )
         )
 
+    def test_linear_time_transformation_scaling(self):
+        """
+        test the scaling of the intervall the function
```

Comment:
General remark, do strings in MeshPy start with an upper case letter and directly after the tripple quotes (no newline).